wiki/Options/en.md

```diff
@@ -1,525 +1,162 @@
 
-![](Options_General_thumb.jpg "fig:Options_General_thumb.jpg") General section
-------------------------------------------------------------------------------
-
-<img src="Options General.jpg" title="Comparison of the Guest and  General section" alt="Comparison of the Guest and  General section" width="300" />
+## General section
 
 ### Sign In
 
 #### Guest
 
-<table style="width:36%;">
-<colgroup>
-<col width="12%" />
-<col width="24%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th><p>Name</p></th>
-<th><p>Effect</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Username</p></td>
-<td><p>Put your registered osu! username here.</p></td>
-</tr>
-<tr class="even">
-<td><p>Password</p></td>
-<td><p>Password for the registered osu! username.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Remember Username</p></td>
-<td><p>Remember your username in your osu!.{username}</p></td>
-</tr>
-<tr class="even">
-<td><p>Remember Password</p></td>
-<td><p>Remember your password in your osu!.{username} and automatically log-in.</p></td>
-</tr>
-</tbody>
-</table>
-
-Click on **Sign in** to to log-in or click on **Create an account** if you do not have the account. Once you have successfully logged-in, it will change as shown below.
-
-#### {username}
-
-`Logged in as {username}`
-
-Click on it to access user stuff.
-
-<table style="width:36%;">
-<colgroup>
-<col width="12%" />
-<col width="24%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th><p>Name</p></th>
-<th><p>Effect</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>1. View Profile</p></td>
-<td><p>View your profile in osu! website.</p></td>
-</tr>
-<tr class="even">
-<td><p>2. Sign Out</p></td>
-<td><p>Sign out in-game.</p></td>
-</tr>
-<tr class="odd">
-<td><p>3. Change Avatar</p></td>
-<td><p>Change your user's profile picture.</p></td>
-</tr>
-<tr class="even">
-<td><p>4. Close</p></td>
-<td><p>Close.</p></td>
-</tr>
-</tbody>
-</table>
+| Name              | Effect                                     |
+|-------------------|--------------------------------------------|
+| Username          | Put your registered osu! username here.    |
+| Password          | Password for the registered osu! username. |
+| Remember Username | Have the client remember your username.    |
+| Remember Password | Have the client remember your password.    |
+
+Click on **Sign in** to to log-in or click on **Create an account** if you do not have the account. Once you have successfully logged-in, your username will display. Clicking on it will display the following:
+
+| Name          | Effect                                 |
+|---------------|----------------------------------------|
+| View Profile  | View your profile on the osu! website. |
+| Sign Out      | Sign out of your account.              |
+| Change Avatar | Change your user's profile picture.    |
+| Close         | Close.                                 |
```

Comment:
Could be more descriptive of what the effect of "Close" will do. In this case, I'd use:
`Close this dialogue.` or something